### ISSUE TITLE
Can I use COCO val mAP metric for paper results?

### ISSUE THREAD

**Issue by sleeplessai** [open-mmlab/mmdetection#2615]
Hi! I am training my model improvement via mmdet toolkit. Is it correct to compare with other states-of-the-arts models under mAP metric on **COCO val set** ? In another word, can I use this COCO val mAP metric for paper results ?

After checking almost data configuration, I find that COCO validation set plays the role of test set to evaluate the model mAP. So I think the performance charts on papers adopt these metrics to compare. 

> data = dict(
>     imgs_per_gpu=2,
>     workers_per_gpu=2,
>     train=dict(
>         type=dataset_type,
>         ann_file=data_root + 'annotations/instances_train2017.json',
>         img_prefix=data_root + 'train2017/',
>         pipeline=train_pipeline),
>     val=dict(
>         type=dataset_type,
>         ann_file=data_root + 'annotations/instances_val2017.json',
>         img_prefix=data_root + 'val2017/',
>         pipeline=test_pipeline),
>     test=dict(
>         type=dataset_type,
>         ann_file=data_root + 'annotations/instances_val2017.json',
>         img_prefix=data_root + 'val2017/',
>         pipeline=test_pipeline))

If not so, I think the true mAP on the COCO test set may submit to [competition page](https://competitions.codalab.org/competitions/5181#participate
) to eval on test set.

Which one I should follow?

**Comment by hellock** [comment]
Results in the modelzoo are reported on val set and you can use it in your paper. If you want to compare with other methods on test-dev, you need to test the checkpoint on test-dev and submit the results to the evaluation server.

**Comment by sleeplessai** [comment]
@hellock 
Thanks. What you mean is I just need to ensure the comparison evaluate on the same, val or test-dev, dataset so that the results would show on my paper are precise.

**Comment by hellock** [comment]
Yes.
